lsp/java/src/main/java/com/itsaky/lsp/java/actions/common/RemoveUnusedImportsAction.java

```diff
@@ -14,71 +14,75 @@
+                .getApplicationContext()
+                .getString(getTitleTextRes());
+    }
 
-  @Override
-  protected int getTitleTextRes() {
-    return R.string.action_remove_unused_imports;
-  }
+    @Override
+    protected int getTitleTextRes() {
+        return R.string.action_remove_unused_imports;
+    }
 
-  @Override
-  public void prepare(ActionData data) {
-    super.prepare(data);
-    if (getVisible()) {
-      if (!hasRequiredData(data)) {
-        markInvisible();
-        return;
-      }
+    @Override
+    public void prepare(ActionData data) {
+        super.prepare(data);
+        if (getVisible()) {
+            if (!hasRequiredData(data, CodeEditor.class)) {
+                markInvisible();
+                return;
+            }
 
-      setVisible(true);
-      setEnabled(true);
+            setVisible(true);
+            setEnabled(true);
+        }
     }
-  }
 
-  @Override
-  public String execAction(ActionData data) {
-    final long start = System.currentTimeMillis();
-    try {
-      CodeEditor editor = requireEditor(data);
-      Content content = editor.getText();
-      if (content != null) {
-        String output = RemoveUnusedImports.removeUnusedImports(content.toString());
-        log.info("Removed unused imports in", System.currentTimeMillis() - start + "ms");
-        return output;
-      }
-    } catch (FormatterException e) {
-      log.error("Failed to remove unused imports", e);
-      return null;
+    @Override
+    public String execAction(ActionData data) {
+        final long start = System.currentTimeMillis();
+        try {
+            CodeEditor editor = requireEditor(data);
+            Content content = editor.getText();
+            if (content != null) {
+                String output = RemoveUnusedImports.removeUnusedImports(content.toString());
+                log.info("Removed unused imports in", System.currentTimeMillis() - start + "ms");
+                return output;
+            }
+        } catch (FormatterException e) {
+            log.error("Failed to remove unused imports", e);
+            return null;
+        }
+        return null;
     }
-    return null;
-  }
 
-  @Override
-  public boolean getRequiresUIThread() {
-    return false;
-  }
+    @Override
+    public boolean getRequiresUIThread() {
+        return false;
+    }
 
-  @Override
-  public void setLabel(String arg0) {}
+    @Override
+    public void setLabel(String arg0) {}
 
-  @Override
-  public void postExec(ActionData data, Object result) {
-    super.postExec(data, result);
-    String text = (String) result;
-    if (text != null) {
-      CodeEditor editor = requireEditor(data);
-      editor.setText(text);
+    @Override
+    public void postExec(ActionData data, Object result) {
+        super.postExec(data, result);
+        if (result != null & result instanceof String) {
+            String text = (String) result;
+            if (text != null) {
```

Comment:
This null check is redundant. If the first condition is true, why would this return false?